### ISSUE TITLE
Eraser Bot source

### ISSUE THREAD

**Issue by jhspetersson** [sethballantyne/eraser-bot-frontend#10]
Hi, do you have Eraser Bot source code by any chance? Thanks!

**Comment by sethballantyne** [comment]
No, I did a search and could only find dead links. If you find it, let me know; it belongs on archive.org. I'll get in touch if I find it.

**Comment by jhspetersson** [comment]
Found Eraser Ultra source on Tastyspleen. There is a mod named NIQ that includes Eraser as well.
